### PR TITLE
zh: update scale tidb-binlog

### LINKED ISSUE #1320: Update the doc of binlog deployment
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
https://github.com/pingcap/tidb-operator/pull/3973 has improved the lifecycle management for Pump so that the doc
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-binlog#%E7%BC%A9%E5%AE%B9%E7%A7%BB%E9%99%A4-pumpdrainer-%E8%8A%82%E7%82%B9 should be updated.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by better0332**
### What is changed, added, or deleted? (Required)

update scale tidb-binlog

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

Fix #1320

### CODE REVIEW

**Comment by DanielZhangQD** [comment]
@july2993 PTAL

**Review by july2993 — APPROVED** [review]

File: zh/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -333,43 +333,27 @@ spec:
...
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl exec binlogctl -n ${namespace} -- /binlogctl -pd-urls "https://${cluster_name}-pd:2379" -cmd offline-pump -node-id ${cluster_name}-pump-${ordinal_id}:8250 -ssl-ca "/etc/binlog-tls/ca.crt" -ssl-cert "/etc/binlog-tls/tls.crt" -ssl-key "/etc/binlog-tls/tls.key"
-    ```
+    其中 `${pump_replicas}` 是你想缩容至的目标副本数。
 
-    然后查看 Pump 的日志输出，输出 `pump offline, please delete my pod` 后即可确认该节点已经成功下线。
+2. 等待 Pump Pod 自动下线被删除
```
Comment (ran-huang) [inline zh/deploy-tidb-binlog.md:346]:
```suggestion
2. 等待 Pump Pod 自动下线被删除。运行以下命令查看 Pod 状态：
```

File: zh/deploy-tidb-binlog.md [hunk 2]
```diff
@@ -333,43 +333,27 @@ spec:
...
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl exec binlogctl -n ${namespace} -- /binlogctl -pd-urls "https://${cluster_name}-pd:2379" -cmd offline-pump -node-id ${cluster_name}-pump-${ordinal_id}:8250 -ssl-ca "/etc/binlog-tls/ca.crt" -ssl-cert "/etc/binlog-tls/tls.crt" -ssl-key "/etc/binlog-tls/tls.key"
-    ```
+    其中 `${pump_replicas}` 是你想缩容至的目标副本数。
 
-    然后查看 Pump 的日志输出，输出 `pump offline, please delete my pod` 后即可确认该节点已经成功下线。
+2. 等待 Pump Pod 自动下线被删除。运行以下命令查看 Pod 状态：
```
[the remark this answers is not in the document]
Comment (better0332) [reply zh/deploy-tidb-binlog.md:346]: watch 命令是看一个变化，我换成一下描述

**Review by ran-huang — APPROVED** [review]

**Pull request merged**